pkg/plugin/plugin.go

```diff
@@ -77,52 +79,62 @@ func NewJiraPlugin(ctx context.Context, cfg *PluginConfig) (*JiraPlugin, error)
 // Validate returns the validation result.
 func (j *JiraPlugin) Validate(ctx context.Context, req *jvspb.ValidateJustificationRequest) (*jvspb.ValidateJustificationResponse, error) {
 	if got, want := req.Justification.Category, jiraCategory; got != want {
-		return nil,
-			fmt.Errorf("failed to perform validation, expected category %q to be %q",
-				got, want)
+		// TODO(https://github.com/abcxyz/jvs-plugin-jira/issues/48): log error details.
```

Comment:
Nit: TODO(#48): log error details. same to everywhere else.

Reply:
Done